crates/bevy_mesh/src/primitives/dim2.rs

```diff
@@ -452,6 +455,28 @@ pub struct RegularPolygonMeshBuilder {
     circumradius: f32,
     sides: u32,
 }
+
+impl RegularPolygonMeshBuilder {
+    /// Creates a new [`RegularPolygonMeshBuilder`] from the radius of a circumcircle and a number
+    /// of sides.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `circumradius` is negative, or if `sides` is less than 3.
```

Comment:
How is +-0 handled?

Reply:
If `circumradius` is 0 (positive or negative), the mesh still builds successfully but is not visible. I don't think it should panic in this case.